### PR TITLE
minor: make inspection chapter in idea xdoc page to be easier to read

### PR DESCRIPTION
**Pull request by romani**

### CODE REVIEW

**Comment by romani** [comment]
failed CIs were restarted

**Comment by romani** [comment]
@rnveach , please do review.

File: src/xdocs/idea.xml [hunk 1]
```diff
@@ -77,18 +77,28 @@
...
         <img alt="Inspections in IntelliJ IDEA" src="images/gui_screenshot_inspections_idea.png"/>
-        <br/><br/>
-        Not all files in repository should be analyzed. For example test input files contain
-        numerous violations on purpose. Our shared inspection scope can be used to exclude such
-        files.<br/>
-        To add custom scope copy file <code>config/intellij-idea-inspection-scope.xml</code>
+        <br/>
+        <br/>
+        ATTENTION: Not all files in repository should be analyzed.
+        For example test input files contain
```
Comment (rnveach) [inline src/xdocs/idea.xml]: Formatting of this line looks off. I thought you split it by periods, but this whole sentence could fit on 1 line, up to the period.
Can you move `numerous violations on purpose.` to this line.
Make `Our shared` all one 1 line too.

Comment (romani) [reply src/xdocs/idea.xml]: done.

**Comment by rnveach** [comment]
> --2018-07-18 04:44:29--  http://download.oracle.com/otn-pub/java/jdk/8u171-b11/512cd62ec5174c3487ac17c61aaa89e8/jdk-8u171-linux-x64.tar.gz?AuthParam=1531889189_62d1d9fdb356546c15f260a5888acaac
Connecting to download.oracle.com (download.oracle.com)|104.106.255.200|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-07-18 04:44:40 ERROR 404: Not Found.
download failed
Oracle JDK 8 is NOT installed.

Travis kicked the bucket.

**Comment by rnveach** [comment]
@romani CI passed.

**Comment by romani** [comment]
squashed , merged

**Pull request closed without merging**